pom.xml

```diff
@@ -62,6 +62,8 @@
         <module>notification-service</module>
         <module>pattern-detection-library</module>
         <module>coverage</module>
+        <module>prebuilt-tasks</module>
+        <module>external-dependencies</module>
```

Comment:
nit: just for semantic, pls move this above `prebuilt-tasks`

Reply:
definitely 'nit'